test/cpp/jit/tests_setup.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+
+def testEvalModeForLoadedModule_setup():
```

Comment:
Is there a reason this file isn't just a straight-line script? I.e. without the function, `main` and `if __name__ ...`, just the contents of `testEvalModeForLoadedModule_setup`

Reply:
`test_cpp_cuda` in test_jit.py also runs `runJITCPPTests()`, and we need to be able to call the test setup code in `test_cpp_cuda` to prepare the `dropout_model.pt` file needed, which is why we need script to be in a Python function.